api/src/paths/codes/all.ts

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { getLogger } from '../../utils/logger';
+import { getAllCodeEntities } from '../../utils/code-utils';
+
+const defaultLog = getLogger('codes/all');
+
+export const GET: Operation = allEntities();
+
+
+GET.apiDoc = {
+  description: 'Fetches a category based on its primary key.',
+  tags: ['category'],
+  responses: {
+    200: {
+      description: 'category get response object array.',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
+            properties: {
+              // Don't specify exact response, as it will vary, and is not currently enforced anyways
+              // Eventually this could be updated to be a oneOf list, similar to the Post request below.
+            }
```

Comment:
We could actually specify the response here, since we know the format of the response from the codes tables, and all of the fields they will include.

Reply:
Defined the response schema with 3 imported object definition files: category, header and code.